Sources/Orbit/Components/InputField.swift

```diff
@@ -15,6 +16,7 @@ public struct InputField: View {
     @Binding private var value: String
     @Binding private var messageHeight: CGFloat
     @State private var isEditing: Bool = false
+    @State private var isSecureTextEntry: Bool = true
 
     let label: String
     let placeholder: String
```

Comment:
The placeholder is displayed below the password value when the field is disabled.